docs/conf.py

```diff
@@ -208,3 +209,4 @@
 
 # Default to using the order defined in source.
 autodoc_member_order = "bysource"
+autodoc_default_options = {}
```

Comment:
This is where i thought maybe we could setup an exclude X class. (specifically for `int`) -- `Duration(int)`